internal/handlers/pg/fetch.go

```diff
@@ -73,3 +74,40 @@ func nextRow(rows pgx.Rows) (*types.Document, error) {
 
 	return doc.(*types.Document), nil
 }
+
+func (h *Handler) protoFetch(ctx context.Context, param fetchParam) ([]*types.Document, error) {
+	var sql string
+	if param.comment != "" {
+		param.comment = strings.ReplaceAll(param.comment, "/*", "/ *")
+		param.comment = strings.ReplaceAll(param.comment, "*/", "* /")
+		param.comment = fmt.Sprintf("/* %s */", param.comment)
+
+		sql = fmt.Sprintf(`SELECT %s _jsonb FROM %s`, param.comment, pgx.Identifier{param.db, param.collection}.Sanitize())
+	} else {
```

Comment:
I think that will be easier to read without `else`. In pseudocode:
```go
sql := `SELECT `
if param.comment != "" {
  // handle /*, */
  sql += `/* ` + params.comment + ` */ `
}
sql += `_jsonb FROM ` + pgx.Identifier{param.db, param.collection}.Sanitize()
```

Reply:
Done.